lib/myfinance/resources/entity.rb

```diff
@@ -34,6 +34,48 @@ def find(entity_id)
           respond_with_object(response, 'entity')
         end
       end
+
+      #
+      # Return attributes from entity
+      #
+      # [API]
+      #   Method: <tt>GET /entities/new</tt>
+      #
+      #   Documentation: https://app.myfinance.com.br/docs/api/entities#get_new
+      #
+      def get_new
+        http.get("/entities/new", body: {}) do |response|
+          respond_with_object(response, 'entity')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.